h/static/scripts/directive/annotation.js

```diff
@@ -655,8 +655,8 @@ function AnnotationController(
   vm.share = function(event) {
     var $container = angular.element(event.currentTarget).parent();
     $container.addClass('open');
-    var shareLinkInput = $container.find('input')[0];
-    shareLinkInput.focus();
+    var shareLinkInput = $container.find('.annotation-share-dialog-link__text');
```

Comment:
CSS classes that exist for use by code rather than just being presentational should have a `js-` prefix or `is-` prefix for classes to indicate a certain state. eg. `js-input` so it is obvious that there is code relying on them. So you would have eg. `<input class="some-dialog__label js-input">` I'm afraid that's a part of the not-yet-written-down style guide for CSS.